src/oemof/solph/_models.py

```diff
@@ -45,7 +45,7 @@ class BaseModel(po.ConcreteModel):
 
     Parameters
     ----------
-    energysystem : EnergySystem object or list of EnergySystem objects
+    energysystem : EnergySystem object or list of EnergySystem objects (experimental)
```

Comment:
```suggestion
    energysystem : EnergySystem object or (experimental) list of EnergySystem objects
```

Maybe this way, to highlight that just the second option is experimental?

Reply:
I'm fine with that! It's easier to understand, indeed. I needed to shorten the line, so I deleted the `of EnergySystem objects` after `list`. It now reads `energysystem : EnergySystem object or list (experimental)` and I would change it to `energysystem : EnergySystem object or (experimental) list`. The explanation that the list needs to hold `EnergySystem` objects moved into the text below. 
Is that okay with you as well?

I apologize for the cumbersome discription.

Reply:
I see. The line `energysystem : EnergySystem object or (experimental) list` sounds reasonable. Maybe also add the keyword in the description, so it reads:

```
    energysystem : EnergySystem object or (experimental) list
        Object that holds the nodes of an oemof energy system graph.
        Experimental: If a list  is passed, the list needs to hold
        EnergySystem objects and a cellular structure is assumed.
        In this case, the first element needs to be the upmost energy cell
        (structurally containing all other cells).
```

Reply:
Good idea! The latest commit includes this change.